index.html

```diff
@@ -859,194 +859,180 @@
+[=context=]. This is particularly true for [=vulnerable=] people, as
+recognising them in different [=contexts=] may force traits that reveal their
+vulnerability into the open. For example, if you meet your therapist at a
+party, you expect them to have different discussion topics with you than they
+usually would, and possibly even to pretend they don't know you.
+
+Note that a UA may not always be powerful enough to prevent [=cross-context
+recognition=]. For example, if two or more services [=inappropriately=]
+require an identifying piece of information in order to use the services.
+
+<dfn>Cross-site recognition</dfn> is when a site determines with high
+probability that a visit to the site comes from the same person as another
+visit to a *different* site.
+
+Traditionally, sites have accomplished this using
+<a href="https://tess.oconnor.cx/2020/10/parties#environment-cross-site">
+cross-site</a> cookies, but it can also be done by having someone navigate to
+a link that has been decorated with an identifier, collecting the same piece
+of identifying information on both sites, or by correlating the timestamps of
+an event that occurs nearly-simultaneously on both sites. A privacy harm
+occurs through [=cross-site recognition=], unless the person could reasonably
+expect the sites to discover this.
+
+<dfn>Same-site recognition</dfn> is when a single site discovers and uses the
+fact that two or more visits probably came from the same [=person=].
+
+This is generally accomplished by either "supercookies" or browser
+[=fingerprinting=]. A privacy harm occurs if a [=person=] reasonably expects
+that they'll be using a different [=identity=] for different visits to a
+single site, but the site engages in [=same-site recognition=].
+
+[=User agents=] can't, in general, determine exactly where [=context=]
+boundaries are within a site, or know how a site allows a [=person=] to
+express that they intend to change [=identities=], so the UA can't enforce
+that sites actually separate [=identities=] at these boundaries.
+
+Instead, [=user agents=] may have to make some assumptions about the borders
+between [=contexts=]. By default, [=user agents=] define a
+<dfn>machine-enforceable context</dfn> or <dfn>partition</dfn> as:
+
+* A set of [=environments=] (roughly iframes (including cross-site iframes),
+workers, and top-level pages)
+* whose [=environment/top-level origins=] are in the [=same site=] (but see
+[[PSL-Problems]])
+* being visited within the same user agent installation (and browser profile,
+container, or   container tab for user agents that support those features)
+* between points in time that the person or user agent clears that [=site=]'s
+cookies and other storage (which is sometimes automatic at the end of each
+session).
+
+Even though this is the default, [=user agents=] can further restrict their
+definition of [=machine-enforceable context=] to better aid their users.
```

Comment:
```suggestion
definition of [=machine-enforceable context=] to better aid their users for instance by secluding identities per subdomain or site path when the structure of a given site is known.
```

Not sure that this is worth adding, I just thought that giving an example might make it more concrete.